src/frontend/http.rs

```diff
@@ -53,7 +72,7 @@ pub async fn start() -> Result<()> {
         // .route("/raw/:bao_hash", get(get_raw))
         .layer(CorsLayer::permissive());
```

Comment:
There are many CorsLayer Options...
Will ::permissinve()   handle all our use cases?
Do we intend to implement more strict arguements?

Reply:
I'd like it to be permissive, it makes it work in more cases, especially in some Nostr clients.